nbconvert/exporters/html.py

```diff
@@ -90,8 +90,11 @@ def process_attachments(self, nb, output):
     def from_notebook_node(self, nb, resources=None, **kw):
         langinfo = nb.metadata.get('language_info', {})
         lexer = langinfo.get('pygments_lexer', langinfo.get('name', None))
-        self.register_filter('highlight_code',
-                             Highlight2HTML(pygments_lexer=lexer, parent=self))
+        highlight_code = Highlight2HTML(pygments_lexer=lexer, parent=self)
```

Comment:
Can consolidate some code here 
```suggestion
        highlight_code = self.filters.get('highlight_code',
                                                           Highlight2HTML(pygments_lexer=lexer, parent=self))
```

Reply:
Good idea, done.